main.cpp

```diff
@@ -52,85 +52,98 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: vending machine
+//  action 1: dispense soda
+vendingMachine.dispenseSoda();
 
-//  action 2:
 
-//  action 3:
- 
+//  action 2: dispense water
+vendingMachine.dispenseWater();
+
+//  action 3: sound alarm
+ vendingMachine.soundAlarm();
+

```

Comment:
Your code should be lined up with the left edge of the editor, since it is not inside of a function
line it up.
- [ ] revise